### ISSUE TITLE
Start a Private Network is a bit inconclusive

### ISSUE THREAD

**Issue by NikVolf** [substrate-developer-hub/substrate-developer-hub.github.io#392]
https://substrate.dev/docs/en/next/tutorials/start-a-private-network/customchain

After last step, it's rather hard to figure out that everything is configured successfully. Should user see configured accounts in the js app?  How can he tell that he now produces blocks, what should he see in node CLI?

**Comment by jimmychu0807** [comment]
Very true, let me add a screenshot of the terminal and what the user should see.

- should have 1 peer instead of 0 peers
- should see proposing block from both Alice's and Bob's terminals.

cc @JoshOrndorff

**Comment by cecton** [comment]
Heya o/

I did this tutorial recently and I have some feedback and recommendations:

1. Adding explicitly in the tutorial "Generating keys" a phrase that says: "We are going to generate at least 2 set keys so we can have at least 2 nodes for the purpose of this tutorial"

I know it's already wrote clearly that we generate keys per node but for someone with limited knowledge I guess it wasn't obvious to me.

2. Adding the instructions on how to start the frontend for every node (in "Creating your private network")

Even if it was written in the README of apps how to start it for a different node (which is not), it's good to give the basic command in the tutorial. But I heard that apparently it's configurable in the settings while the frontend is running.

3. At the end I wasn't sure what to expect. So I think I succeeded when I saw that the 2 nodes had logs increasing values similar to the previous tutorial

I hope this helps
